### PR TITLE
Mark style as HTML safe

### LINKED ISSUE #222: List-View generates warnings
Using list-view from pr-219-fix branch with ember 1.11, while accessing any view with a list I get the following warning:

"Binding style attributes may introduce cross-site scripting vulnerabilities; please ensure that values being bound are properly escaped. For more information, including how to disable this warning, see http://emberjs.com/deprecations/v1.x/#toc_warning-when-binding-style-attributes."

Changing ```list-view-mixin.js``` to return ```(style).htmlSafe()``` cleans one of the forty something warnings i'm getting.

### PR DESCRIPTION
**Pull request by pedrokiefer**
Should fix #222, as I think it's safe to assume that those styles won't do any harm.

### CODE REVIEW

File: addon/list-item-view-mixin.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function positionElement() {
 
 export default Ember.Mixin.create({
   classNames: ['ember-list-item-view'],
-  style: '',
+  style: ('').htmlSafe(),
```
Comment (stefanpenner) [inline addon/list-item-view-mixin.js]: this is a prototype extension, which means it wont work for users with those disabled, please use the long-form version

**Comment by pedrokiefer** [comment]
@stefanpenner PR updated as requested.

**Comment by stefanpenner** [comment]
@pedrokiefer tests appear to be failing

**Comment by pedrokiefer** [comment]
My bad, it was missing a parentheses!

**Comment by stefanpenner** [comment]
released as 0.0.6

**Pull request merged**